### ISSUE TITLE
SummationDecapode with multiple summations not drawing correctly

### ISSUE THREAD

**Issue by GeorgeR227** [AlgebraicJulia/Decapodes.jl#72]
I tried creating a SummationDecapode with multiple summations and I noticed that while the decapode was parsed correctly, since the sum variable names are called the same name, the actual drawing seems to be off. We expect that Σ2 should be pointing to the sum on the right instead of the one on the left.

```Julia
DecaTest = quote
  A::Form0{X}
  B::Form0{X}
  C::Form0{X}
  D::Form0{X}

  D == k(A + B) + p(C + B)
end

Test = SummationDecapode(parse_decapode(DecaTest))

julia> Test
SummationDecapode{Any, Any, Symbol} with elements Var = 1:8, TVar = 1:0, Op1 = 1:2, Op2 = 1:0, Σ = 1:3, Summand = 1:6
┌─────┬───────┬──────┐
│ Var │  type │ name │
├─────┼───────┼──────┤
│   1 │ Form0 │    A │
│   2 │ Form0 │    B │
│   3 │ Form0 │    C │
│   4 │ Form0 │    D │
│   5 │ infer │   •1 │
│   6 │ infer │  sum │
│   7 │ infer │   •2 │
│   8 │ infer │  sum │
└─────┴───────┴──────┘
┌─────┬─────┬─────┬─────┐
│ Op1 │ src │ tgt │ op1 │
├─────┼─────┼─────┼─────┤
│   1 │   6 │   5 │   k │
│   2 │   8 │   7 │   p │
└─────┴─────┴─────┴─────┘
┌───┬─────┐
│ Σ │ sum │
├───┼─────┤
│ 1 │   6 │
│ 2 │   8 │
│ 3 │   4 │
└───┴─────┘
┌─────────┬─────────┬───────────┐
│ Summand │ summand │ summation │
├─────────┼─────────┼───────────┤
│       1 │       1 │         1 │
│       2 │       2 │         1 │
│       3 │       3 │         2 │
│       4 │       2 │         2 │
│       5 │       5 │         3 │
│       6 │       7 │         3 │
└─────────┴─────────┴───────────┘
```

![image](https://user-images.githubusercontent.com/78235421/196505461-64b2296e-ac40-4b05-b19e-532761865ed4.png)

**Comment by lukem12345** [comment]
@GeorgeR227 This is now priority number 1. Ahead of rewrite averaging!

**Comment by GeorgeR227** [comment]
Managed to find the issue. It's what I expected, in that there's some matching on names and since anonymous sums are all called "sum:Ω•", then it's finding multiple sums. I've fixed the error checking for this so the code should at least notice the issue but I'll keep working on patching it.

**Comment by lukem12345** [comment]
Closed by #75
